tests_pytest/pytorch_tests/e2e_tests/test_quantization_preserving_holder.py

```diff
@@ -0,0 +1,133 @@
+
+def get_tpc():
+    base_config = schema.OpQuantizationConfig(
+            default_weight_attr_config=AttributeQuantizationConfig(),
+            attr_weights_configs_mapping={},
+            activation_quantization_method=QuantizationMethod.POWER_OF_TWO,
+            activation_n_bits=8,
+            supported_input_activation_n_bits=8,
+            enable_activation_quantization=True,
+            quantization_preserving=False,
+            signedness=Signedness.AUTO)
+    
+    default_config = schema.OpQuantizationConfig(
+            default_weight_attr_config=AttributeQuantizationConfig(),
+            attr_weights_configs_mapping={},
+            activation_quantization_method=QuantizationMethod.POWER_OF_TWO,
+            activation_n_bits=8,
+            supported_input_activation_n_bits=8,
+            enable_activation_quantization=True,
+            quantization_preserving=False,
+            signedness=Signedness.AUTO)
+
+    mixed_precision_cfg_list = [base_config]
+    default_configuration_options = schema.QuantizationConfigOptions(quantization_configurations=tuple([default_config]))
+    mixed_precision_configuration_options = schema.QuantizationConfigOptions(quantization_configurations=tuple(mixed_precision_cfg_list),
+                                                                             base_config=base_config)
+
+    operator_set = []
+    preserving_quantization_config = (default_configuration_options.clone_and_edit(enable_activation_quantization=False, quantization_preserving=True)
+                              .clone_and_edit_weight_attribute(enable_weights_quantization=False))
+    operator_set.append(schema.OperatorsSet(name=schema.OperatorSetNames.DROPOUT, qc_options=preserving_quantization_config))
+    operator_set.append(schema.OperatorsSet(name=schema.OperatorSetNames.FLATTEN, qc_options=preserving_quantization_config))
+
+    conv = schema.OperatorsSet(name=schema.OperatorSetNames.CONV, qc_options=mixed_precision_configuration_options)
+    fc = schema.OperatorsSet(name=schema.OperatorSetNames.FULLY_CONNECTED, qc_options=mixed_precision_configuration_options)
+    operator_set.extend([conv, fc])
+
+    tpc = schema.TargetPlatformCapabilities(
+        default_qco=default_configuration_options,
+        operator_set=tuple(operator_set))
+    return tpc
+
+test_input_0 = ("conv1_activation_holder_quantizer",)
+test_input_1 = ("conv2_activation_holder_quantizer",)
+test_input_2 = ("dropout_activation_holder_quantizer",)
+test_input_3 = ("flatten_activation_holder_quantizer",)
+test_input_4 = ("fc_activation_holder_quantizer",)
+
+test_expected_0 = (PytorchActivationQuantizationHolder,)
+test_expected_1 = (PytorchActivationQuantizationHolder,)
+test_expected_2 = (PytorchPreservingActivationQuantizationHolder,)
+test_expected_3 = (PytorchPreservingActivationQuantizationHolder,)
+test_expected_4 = (PytorchActivationQuantizationHolder,)
+
+@pytest.mark.parametrize(("inputs", "expected"), [
```

Comment:
No point in running this test multiple times with the same model. Better to check akk the QuantizationHolders explicitly after the MCT.